index.html

```diff
@@ -0,0 +1,197 @@
+        <q class="fs-5 fw-semibold"
+          >Hi noobs! You don't want to miss this one!</q
+        >
+        <h1 class="fw-bold text-uppercase">🦄Unicorn Con 2023🧚</h1>
+        <p
+          class="p-3 bg-dark fw-bolder opacity-75 text-light border border-white border-opacity-10"
+        >
+          <span class="fw-bold fs-5">Kohai</span> inc. just surpassed itself
+          this time. An event without precedence will take place this year in
+          West Blue, Ohara of all places. Put on your favorite cosplay costume,
+          because we will bazinga like it is our last day.
+        </p>
+        <p class="fs-1 fw-bold text-light">Mon 29 - Wen 31 May</p>
+        <p class="text-light fw-semibold">@Colosseum Ohara West Blue</p>
+      </div>
+    </header>
+    <main>
+      <section id="program" class="d-flex flex-column align-items-center pb-3">
+        <h2 class="text-center fw-bolder pt-5 text-light">Main Program</h2>
+        <span class="line-primary"></span>
+        <div
+          class="d-flex flex-column flex-md-row flex-wrap justify-content-md-center m-3"
+        >
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-solid fa-masks-theater fs-1 p-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Perfomances</h3>
+            <p class="p-1">
+              A lot of performers will join us at that time to blow our mind.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-solid fa-pen-to-square fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Autographs</h3>
+            <p class="p-1">
+              All your favorite authors will gather to make your dream a
+              reality.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-solid fa-handshake fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">
+              Cosplayer meet&greet
+            </h3>
+            <p class="p-1">
+              Every big cosplayers from all over the world will be there.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-sharp fa-solid fa-star fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Special events</h3>
+            <p class="p-1">
+              The "gear fourth" show will be the wild card but more special
+              events will also take place.
+            </p>
+          </div>
+        </div>
```

Comment:
Your program section should contain five cards instead of four with no spaces between them. Kindly check and rectify this anomalies. 

Your design 

![Screenshot from 2023-01-12 08-25-14](https://user-images.githubusercontent.com/65233860/212016854-231fdf26-dd6e-4406-ab79-6c4b8bd2b295.png)


Actual design

![Screenshot from 2023-01-12 08-25-30](https://user-images.githubusercontent.com/65233860/212016866-3372c0ca-2640-4eb3-ba2c-f223d78090e1.png)